extensions/cornerstone/src/commandsModule.js

```diff
@@ -318,6 +322,11 @@ const commandsModule = ({ servicesManager }) => {
       storeContexts: [],
       options: {},
     },
+    clearMeasurements: {
+      commandFn: actions.clearMeasurements,
+      storeContexts: [],
+      options: {},
+    },
```

Comment:
Ha. That's funny. I guess this is technically easier to do with a command. Can you move this to the `default` extension's commands, as this works for clearing measurements regardless of ext?